src/Microsoft.Health.Fhir.Liquid.Converter/Ccda/CcdaDataParser.cs

```diff
@@ -18,7 +18,7 @@ public static class CcdaDataParser
     {
         public static IDictionary<string, object> Parse(string document)
         {
-            if (string.IsNullOrEmpty(document))
+            if (string.IsNullOrWhiteSpace(document))
```

Comment:
The error code `NullOrEmptyInput` is used to handle `IsNullOrEmpty` and `IsNullOrWhiteSpace` simultaneously, maybe we can split the error code to two types or rename it to a more general name.

Reply:
Thanks, the error code and message are unified to `NullOrWhiteSpaceInput` to handle `string.IsNullOrWhiteSpace()` only.